src/aggregated/account_details.rs

```diff
@@ -0,0 +1,64 @@
+use actix::Addr;
+
+use near_client::{Query, ViewClientActor};
+use near_indexer::near_primitives::hash::CryptoHash;
+use near_indexer::near_primitives::types::{Balance, BlockId, BlockReference};
+use near_indexer::near_primitives::views::{AccountView, QueryRequest, QueryResponseKind};
+
+pub(crate) async fn get_account_balance(
+    view_client: &Addr<ViewClientActor>,
+    account_id: &str,
+    block_height: u64,
+) -> Result<Balance, String> {
+    get_account_view_for_block_height(view_client, account_id, block_height)
+        .await
+        .map(|acc| acc.amount)
+}
+
+pub(crate) async fn get_contract_code_hash(
+    view_client: &Addr<ViewClientActor>,
+    account_id: &str,
+    block_height: u64,
+) -> Result<CryptoHash, String> {
+    get_account_view_for_block_height(view_client, account_id, block_height)
+        .await
+        .map(|acc| acc.code_hash)
+}
+
+async fn get_account_view_for_block_height(
```

Comment:
It's enough to have only one call to this function, understood it only now
Will fix it tomorrow

Reply:
UPD: I call them for different accounts, so everything is correct. I need balance for foundation accs, and I need code hash for lockups. False alarm 😄